metrics/network/network-metrics-iperf3.sh

```diff
@@ -21,9 +21,15 @@
 # - bandwith duplex
 # - jitter
 #
-# These metrics/results will be got from the interconnection between a container
-# server and client using the iperf tool.
-# container-server <----> container-client
+# These metrics/results will be got from the interconnection between
+# a client and a server using iperf tool.
+# The following cases are covered:
+#
+# case 1:
+#  container-server <----> container-client
```

Comment:
@MarioCarrilloA looks like a typo here. I assume you meant host-server

Reply:
This script includes multiple network tests using iperf with different configurations, one of them is `container-server<->container-client` and I have added some of them where the server is the container  `container-server<->host-client` there are not yet cases about `host-server<->container-client`